tests/readPreference/readpreference-var_export-001.phpt

```diff
@@ -0,0 +1,78 @@
+--TEST--
+MongoDB\Driver\ReadPreference: var_export()
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$tests = [
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY),
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY_PREFERRED),
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY),
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY_PREFERRED),
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_NEAREST),
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY, []),
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, [['dc' => 'ny']]),
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, [['dc' => 'ny'], ['dc' => 'sf', 'use' => 'reporting'], []]),
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, null, ['maxStalenessSeconds' => 1000]),
+];
+
+foreach ($tests as $test) {
+    echo var_export($test, true), "\n";
+}
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECT--
+MongoDB\Driver\ReadPreference::__set_state(array(
+   'mode' => 'primary',
+))
+MongoDB\Driver\ReadPreference::__set_state(array(
+   'mode' => 'primaryPreferred',
+))
+MongoDB\Driver\ReadPreference::__set_state(array(
+   'mode' => 'secondary',
+))
+MongoDB\Driver\ReadPreference::__set_state(array(
+   'mode' => 'secondaryPreferred',
+))
+MongoDB\Driver\ReadPreference::__set_state(array(
+   'mode' => 'nearest',
+))
+MongoDB\Driver\ReadPreference::__set_state(array(
+   'mode' => 'primary',
+))
+MongoDB\Driver\ReadPreference::__set_state(array(
+   'mode' => 'secondary',
+   'tags' => 
+  array (
```

Comment:
Does `var_export()` actually indent with 2 spaces here, vs. 3 spaces for the fields above (e.g. "tags")? Odd.

Reply:
It adds a tab instead of 3 spaces, but otherwise: yes. It also adds a trailing space in some places.

Reply:
Based on the following:

```php
var_export((object)['foo'=>['bar']]);
```

We get:

```
stdClass::__set_state(array(
   'foo' => 
  array (
    0 => 'bar',
  ),
))
```

The output is all spaces (thankfully no mixed tabs/spaces), but it's definitely standard behavior.